### DISCUSSION TITLE
Add hyperscript link to the main index.html page

### DISCUSSION THREAD

**Discussion by rickhall** [MithrilJS/mithril.js#2923] [Ideas]
I started delving into Mithril yesterday and find it very interesting. As part of my reading, I read through the "Key concepts" listed on the front page one after the other. However, I was left with some questions until @osban pointed me to the hyperscript page at https://mithril.js.org/hyperscript.html.

Funnily enough, one of the question I had while reading the Vnodes page was, "I wonder what the signature of m() is?", which is the first thing covered on the hyperscript page and the hyperscript page is not linked from any of the "Key concept" pages.

I think it would make a lot of sense to add a link to "Hyperscript" after "Vnodes", in the "Key concepts" section in index.html, since it logically follows from the Vnodes material and alludes to the upcoming other key concepts.

Otherwise, keep up the excellent work. Thanks.

**Comment by tbreuss** [comment]
It seems that this issue is still relevant.

@dead-claudia I would love to fix this. Would you mind assign me the issue?

**Comment by dead-claudia** [comment]
Sorry @pygy for the noise - somehow crossed up usernames.
